src/test/scala/ai/privado/languageEngine/csharp/dataflow/DataflowTests.scala

```diff
@@ -30,4 +46,60 @@ class DataflowTests extends CSharpFrontendTestSuite {
       sink.reachableByFlows(src).size shouldBe 1
     }
   }
+
+  "dataflow for derived sources tagged using member name" should {
+    val cpg = code("""
+        |namespace Foo;
+        |
+        |class Person {
+        |   public String firstName { get; set; }
+        |   public String lastName { get; set; }
+        |}
+        |
+        |class Runner {
+        | static void Main() {
+        |   Person p = new Person();
+        |   Console.WriteLine(p);
+        |
+        |   var userPassword = 123;
+        |   Console.WriteLine(userPassword);
+        |}
+        |}
+        |""".stripMargin)
+      .withRuleCache(ruleCache)
+      .withPrivadoInput(privadoInput)
+      .withAuditCache(auditCache)
+      .withAppCache(appCache)
+
+    val dataflowMap =
+      Dataflow(cpg, new StatsRecorder()).dataflow(privadoInput, ruleCache, dataflowCache, auditCache, appCache)
```

Comment:
Refactor the Test cases, to do validation on the Privado json Map. Ex- use `cpg.getPrivadoJson`